From eb716fe8a06ea32c6d309e97d92a3fed6ad43699 Mon Sep 17 00:00:00 2001
From: m34782 <lewis.hanly@microchip.com>
Date: Wed, 1 Jul 2020 03:14:49 +0100
Subject: [PATCH] Polarfire SoC DTS support

Adding: MSS-GPIO/I2C/SPI/qSPI/CAN-UIO/USB/UIO/PCIE


Signed-off-by: Lewis Hanly <lewis.hanly@microchip.com>
---
 arch/riscv/boot/dts/sifive/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/riscv/boot/dts/sifive/Makefile b/arch/riscv/boot/dts/sifive/Makefile
index 6d6189e6e4af..0386be097a29 100644
--- a/arch/riscv/boot/dts/sifive/Makefile
+++ b/arch/riscv/boot/dts/sifive/Makefile
@@ -1,2 +1,3 @@
 # SPDX-License-Identifier: GPL-2.0
 dtb-$(CONFIG_SOC_SIFIVE) += hifive-unleashed-a00.dtb
+dtb-$(CONFIG_POLARFIRE_SOC) += mpfs.dtb
